templates/footer.mustache

```diff
@@ -0,0 +1,9 @@
+<footer>
+  <ul>
+    <li><a href="//mozilla.com"><img src="mozilla-logo.png"></a>
+    <li><a href="//blog.mozilla.org">Blog</a>
+    <li><a href="//mozilla.org/privacy">Privacy Policy</a>
+    <li><a href="//mozilla.org/cookies">Cookies</a>
+    <li><a href="//mozilla.org/legal">Legal Notices</a>
+  </ul>
+</footer>
```

Comment:
I'm confused. What's "templates/footer.mustache" used for when this same code is duplicated in src/index.html:22-30?